### DISCUSSION TITLE
Halo 文档中对于 NPM 转发容器内网络的问题

### DISCUSSION THREAD

**Discussion by qksuki** [halo-dev/halo#7044] [General]
原文：https://docs.halo.run/getting-started/install/other/nginxproxymanager/

其中提到：

> 因为样例的 NPM 和 Halo 搭建在同一台 VPS 上，所以这边的 IP，图中填的是 172.17.0.1，为 Docker 容器内部的 IP 地址，\
> 可以通过下面的命令查询：
> ```sh
> ip addr show docker0
> 4: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default 
>  link/ether 02:42:e4:a3:b5:b9 brd ff:ff:ff:ff:ff:ff
>     inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
>      valid_lft forever preferred_lft forever
> ```
> 这边的 IP 是 172.17.0.1，填入这个 IP，可以不用打开防火墙的 8090 端口。
> 当然，如果你的 NPM 和 Halo 不在同一台服务上，你需要在 IP 部分填入 你的 Halo 所在的服务器的 IP，并在服务商（部分服务商如腾讯、阿里）的后台打开 8090 端口。

但是我测试后发现，直接访问 docker0 的网络 (172.17.0.1) 并不能达成目标。事实上，我只有访问 docker halo 容器的 ip 地址 (172.18.0.3)，才能实现转发定向。

使用 docker 部署时，halo 的默认网络是 `halo_halo_network`，网关为 `172.18.0.1`，halo 容器为 `172.18.0.3`，这和文档中提到的 docker0 的网络地址描述不是很相符。也许描述有歧义？

**Comment by ruibaby** [comment]
其实并不推荐通过这种方式去配置网络，建议是创建一个 Network，并配置到 Nginx Proxy Manger 和 Halo 的容器中。如果你有兴趣的话，可以帮忙改一下。

**Reply by qksuki** [reply]
我已经提交了相关的 PR，并且做了相应的测试，有问题可以和我说下。
> https://github.com/halo-dev/docs/pull/473
